LondonBritishLibrary/orient/BLorient691.xml

```diff
@@ -54,67 +239,101 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                 </support>
                                 <extent>
                                     <measure unit="leaf">243</measure>
-                                    <dimensions type="outer" unit="mm">
-                                        <height>365</height>
-                                        <width>235</width>
+                                    <dimensions type="outer" unit="in">
+                                        <height>14.0375</height>
+                                        <width>9.25</width>
                                     </dimensions>
                                 </extent>
-                                
-                                
+
+
                             </supportDesc>
-                            
+
                             <layoutDesc>
                                 <layout columns="2" writtenLines="22 26" cert="high"/>
                             </layoutDesc>
                         </objectDesc>
-                        
+
                         <handDesc>
-                            
+
                             <handNote script="Ethiopic" xml:id="h1">
-                                <desc>The manuscript is written in a fine  hand, of the <date>15th century</date>. 
-                                    
+                                <desc>The manuscript is written in a fine hand, of the <date>15th century</date>.
+
                                 </desc>
-                                
+
                                 <date notBefore="1400" notAfter="1500" resp="PRS10171Wright"/>
-                                
+
                             </handNote>
-                            
-                        </handDesc>   
+
+                        </handDesc>
                         <additions>
-                            
-                            
+<list>
+<item xml:id="a1">
+<locus target="#218v"/>
+<desc type="PurchaseNote">ለዝንቱ፡ መጽሐፍ፡ አጥረይክዎ፡ አነ፡ <persName ref="PRS13146SaggaKrestos">ጸጋ፡ ክርስቶስ፡</persName> በ፫ሸማ፡ ወ፬አሞሌ፡
+እምወለተ፡ አባ፡ <persName ref="PRS13147HablaSellase">ሐብለ፡ ሥላሴ፡</persName> ዘ<placeName ref="LOC1602Atrons">አትሮንሰ፡ ማርያም</placeName>፡፡
+መድኅንየሂ፡ <persName ref="PRS13172Ars">አርሳርዮስ፡</persName> ዘአትሮንሰ፡ ማርያም፡፡ ወመሳክርትሂ፡ በዥርወንድ፡
+<persName ref="PRS13165KeflaGiyorgis">ክፍለ፡ ጊዮርጊስ፤</persName> <persName ref="PRS13166TasfaIyasus">ተስፋ፡ ኢየሱስ፡</persName>
+ጸሓፈ፡ <persName ref="PRS13167Baselyos">ባስልዮስ፡</persName> አውቂያየ፡ ወዓሌ፡ አዛዥ፡ <persName ref="PRS13168Demyanos">ድምያኖስ፡</persName>
+ <persName ref="PRS13169YamanaKrestos">የማነ፡ ክርስቶስ፡</persName> አቤቶ፡ <persName ref="PRS13170EdaKrestos">እደ፡ ክርስቶስ፡</persName> ዘዋረብ፡፡</desc>
```

Comment:
the role could be added to the personal names where possible

Reply:
Other than owner, the role of these people is not clear to me.